### ISSUE TITLE
Write/code from prompt -- producing incomplete code

### ISSUE THREAD

**Issue by jme6f4** [MichelNivard/gptstudio#15]
Hi, Thank you for making this package available. 

I'm having an issue with the Write/code from prompt function. When I apply the comment below to the write/code function, it produces the following, partial code. 

```
# Write R code to make a scatterplot with ggplot2 make the points blue

> library(ggplot2)
> 
> ggplot(data = mtc

```
However, when I apply the "Comment your code" function, it seems to work fine. 

```
> library(ggplot2) #load ggplot2 package
> 
> ggplot(data = mtcars, aes(x = wt, y = mpg)) + #create a scatterplot of mpg vs wt
>   geom_point(color = "blue") #make the points blue
> 
> mean(mtcars$mpg) #calculate the mean of mpg
```

I attempted to apply the comments provided here to the Write/code function, but the function seems to be running into the same issue. 

First attempt (direct copy/paste of comments above)
```
# create a scatterplot of mpg vs wt make the points blue
> 
> plt.scatter(df['wt'], df['mp
```

Second attempt (specifying ggplot2)
```
# create a scatterplot using ggplot2 of mpg vs wt make the points blue
> 
> ggplot(data = mtcars, aes(x =
```

Might this be a problem with my working environment? I've tried to search for an issue that might be causing this, but apologies if I'm missing something that should be obvious.

**Comment by JamesHWade** [comment]
Hi @jme6f4, glad that you are trying out the pacakge. I think you've found a bug! The number of tokens being passed looks like it's not being respected. Let me dig in a see if I can find a quick fix.

**Comment by JamesHWade** [comment]
The should be fixed with #16. Please let me know if it does not.

**Comment by jme6f4** [comment]
Thanks!! Your update may have fixed it, or it may have been user error on my part. In case someone else has this issue and sees this thread.... make sure to set your API key with quotes around it (see below). Either way, thanks -- this is has already been very useful.

`OPENAI_API_KEY="Your.API.Key.Here"`
